dev/check_files.py

```diff
@@ -86,66 +83,80 @@ def create_docker(txt: str):
-        print(p)
+def strip_rc_suffix(version):
+    return re.sub(r'rc\d+$', '', version)
 
-        name = name_tpl.format(p.replace(".", "_"), version)
-        # Check sources
-        check_all_present("sources", filter_files(files, name))
 
-        # Check wheels
-        name = name.replace("_", "-")
-        if check_all_present("wheel", filter_files(files, name)):
-            pip_packages.append(f"{name.rpartition('-')[0]}=={version}")
+def print_status(file, is_found: bool):
+    color, status = ('green', 'OK') if is_found else ('red', 'MISSING')
+    print(f"    - {file}: [{color}]{status}[/{color}]")
 
-    return pip_packages
 
+def check_all_files(actual_files, expected_files):
+    missing_list = []
+    for file in expected_files:
+        is_found = file in actual_files
+        if not is_found:
+            missing_list.append(file)
+        print_status(file=file, is_found=is_found)
+    return missing_list
 
-def check_release(files: List[str], version: str):
-    print(f"apache_airflow-{version}")
 
-    # Check bin
-    name = f"apache-airflow-{version}-bin"
-    check_all_present("binaries", filter_files(files, name))
+def check_release(files: List[str], version: str):
+    print(f"Checking airflow release for version {version}:\n")
+    version = strip_rc_suffix(version)
+
+    expected_files = expand_name_variations(
+        [
+            f"apache-airflow-{version}.tar.gz",
+            f"apache-airflow-{version}-source.tar.gz",
+            f"apache_airflow-{version}-py3-none-any.whl",
+        ]
+    )
+    return check_all_files(expected_files=expected_files, actual_files=files)
 
-    # Check sources
-    name = f"apache-airflow-{version}-source"
-    check_all_present("sources", filter_files(files, name))
 
-    # Check wheels
-    name = f"apache_airflow-{version}-py"
-    check_all_present("wheel", filter_files(files, name))
+def expand_name_variations(files):
+    return list(sorted(base + suffix for base, suffix in product(files, ['', '.asc', '.sha512'])))
 
 
 def check_upgrade_check(files: List[str], version: str):
-    print(f"apache_airflow-upgrade-check-{version}")
+    print(f"Checking upgrade_check for version {version}:\n")
+    version = strip_rc_suffix(version)
+
+    expected_files = expand_name_variations(
+        [
+            f"apache-airflow-upgrade-check-{version}-bin.tar.gz",
+            f"apache-airflow-upgrade-check-{version}-source.tar.gz",
+            f"apache_airflow_upgrade_check-{version}-py2.py3-none-any.whl",
+        ]
+    )
+    return check_all_files(expected_files=expected_files, actual_files=files)
 
-    name = f"apache-airflow-upgrade-check-{version}-bin"
-    check_all_present("binaries", filter_files(files, name))
 
-    name = f"apache-airflow-upgrade-check-{version}-source"
-    check_all_present("sources", filter_files(files, name))
+def warn_of_missing_files(files):
+    print(f"[red]Check failed. Here are the files we expected but did not find:[/red]\n")
```

Comment:
```suggestion
    print("[red]Check failed. Here are the files we expected but did not find:[/red]\n")
```

Should fix failing static check:

```
dev/check_files.py:156:11: F541 f-string is missing placeholders
```

Reply:
sorry fixed before saw you did the work